src/webserver/DOSGuard.h

```diff
@@ -84,17 +415,65 @@ class BasicDOSGuard : public BaseDOSGuard
         sweepHandler.setup(this);
     }
 
+    /**
+     * @brief Checks if IP address is malformed
+     *
+     * @param ip IP address to check
+     */
+    [[nodiscard]] bool
+    isValidIPMixed(std::string const& ip) const
```

Comment:
This function can be removed. We don't need to call it in isWhiteListed

Reply:
@cindyyan317 removing because of your assertion that input ip will never be malformed